tests/s2_inference/test_clip_onnx_utils.py

```diff
@@ -7,7 +7,7 @@
 from unittest import mock
 import requests
 # NOTE: circular reference between model_registry & onnx_clip_utils
-import marqo.s2_inference.model_registry as model_registry
+import marqo.s2_inference.model_loaders as model_registry
```

Comment:
this is confusing. it's better to `import marqo.s2_inference.model_loaders as model_loaders`